libs/plugins/flow-client/src/lib/shared/dynamic-form/object/object-type.component.html

```diff
@@ -12,7 +12,6 @@
     [formControl]="fieldGroup.get('value')"
     [formValueWriteTransformerFn]="transformValueIn"
     [formValueChangeTransformerFn]="transformValueOut"
-    [shouldUpdateValue]="shouldUpdateEditorValue"
```

Comment:
Removing this logic actually created a cosmetic issue which can be tracked separately. I could not include the fix in this as I need more time to find a trivial solution. 

The steps to reproduce cosmetic issue:
1. Test run the flow from the beginning
2. Open the debug panel
3. Select one tile from the diagram and notice that tile's output is reflected in the debug panel
4. Select another tile from the diagram keeping the debug panel open
5. Notice that the newly selected tile's output reflected in the debug panel
6. Now when you switch to other tile, notice that the tile visual changes as if it is not ran. But the executed outputs are displayed as per the expectation
<img src="https://user-images.githubusercontent.com/23206463/72920103-a3ade180-3d6e-11ea-81e9-bce3ea637a06.png" width="200px">

--------------------
<img src="https://user-images.githubusercontent.com/23206463/72920172-c8a25480-3d6e-11ea-8819-fe5ece16b71f.png" width="300px">